packages/babel-runtime/helpers/esm/iterableToArray.js

```diff
@@ -1,3 +1,11 @@
 export default function _iterableToArray(iter) {
-  if (Symbol.iterator in Object(iter) || Object.prototype.toString.call(iter) === "[object Arguments]") return Array.from(iter);
-}
\ No newline at end of file
+  if (
+    Array.isArray(iter)
+    || typeof iter === 'string'
+    || (typeof Symbol === 'function' && Symbol.iterator in Object(iter))
+    || (iter && 'length' in iter)
```

Comment:
Array.from handles non-iterable arraylike objects, so this seems like it fixes a bug unrelated to “there’s no Symbol”

Reply:
true, i updated the commit to reflect this